plugin/trino-bigquery/pom.xml

```diff
@@ -225,6 +220,12 @@
             <artifactId>avro</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>log-manager</artifactId>
+            <scope>runtime</scope>
```

Comment:
Why needed explicitly here? `runtime`  should be rarely needed, should be pulled by a dependency

Reply:
It is referenced directly by `BigQueryQueryRunner`. Add the standard comment
```xml
<!-- used by tests but also needed transitively -->